error.go

```diff
@@ -27,19 +28,26 @@ func NewAPIError(region string, service string, e error) Err {
 	}
 }
 
-// Error rerturns a string which summarize how many errors happened, in which regions and for which services.
+// Error returns a string which summarize how many errors happened and for
+// each error, the region, the service and  the error message reported by
+// AWS original error.
 func (e Errs) Error() string {
-	var output [][]string
+	if len(e) == 0 {
+		return ""
+	}
+
+	var details []string
 
 	for _, err := range e {
-		output = append(output, []string{err.Region(), err.Service()})
+		details = append(details, err.Error())
 	}
-	return fmt.Sprintf("%d error(s) occured: %s", len(e), output)
+
+	return fmt.Sprintf("%d error(s) occurred.\n\t%s", len(e), strings.Join(details, "\n\t"))
 }
 
 // Error returns a string containing the region, service as well as the original API error message.
 func (e *callErr) Error() string {
-	return fmt.Sprintf("%s: error while using '%s' service - %s",
+	return fmt.Sprintf(`region: %s, service: %s, Error message: "%s"`,
```

Comment:
I'm not sure on why having both `<backtick>` and `<quote>` within this Sprintf?

Reply:
Because I didn't want to escape the `"` that the string has.
If you prefer to use `"` then I can replace it by: `"region: %s, service: %s, Error message: \"%s\""`

Reply:
Isn't it the purpose of `%q`? https://play.golang.org/p/1o6owwrO5dP

Reply:
it looks like yes, I didn't remember it. I'll replace it